src/bento/components/jwplayer/1.0/bento-jwplayer.css

```diff
@@ -0,0 +1,17 @@
+/*
+ * Pre-upgrade:
+ * - display:block element
+ * - size-defined element
+ */
+ bento-jwplayer {
```

Comment:
Eventually, we'd like to prevent duplicates of this file. We'll likely have to inverse the current model, where AMP transforms the Bento version of this file to create its own.

Reply:
Yeah, that is exactly what I would like to do